docs/book/src/introduction/core_library.md

```diff
@@ -0,0 +1,48 @@
+# Core Library
+
+The Sway Core Library, like the name suggests contains core operators and logic for the primitive types of the Sway programming language. These traits and methods are an extension of the [primitive types](https://docs.fuel.network/docs/sway/basics/built_in_types/#primitive-types) `u8`, `u16`, `u32`, `u64`, `u256`, `str[]`, `str`, `bool` and , `b256` and can be used where appropriate.
+
+The latest core library documentation can be found in the [Core Library Book](https://fuellabs.github.io/sway/master/core/). If the latest version is not compatible please refer to the appropriate tagged release.
+
+## Using the Core Library
+
+Core library functionalities do not need to be explicitly imported and will work out of the box after creating any new Sway project with [`forc new`](../forc/commands/forc_new.md). The `use` keyword is simply not required.
+
+Consider this example of using the modulo function for two like value types:
+
+```sway
+let val_1 = 10;
+let val_2 = 2;
+let result = val_1 % val_2;
+```
+
+Developers don't need to concern themselves with underflows and overflows because the Sway compiler automatically manages these issues during execution.
+
+## Core Library Prelude
+
+Sway core operations and logic are limited to their respective types. In other words, intuitively, the add `+` operation will be available for all unsigned integers in Sway but not for booleans.
```

Comment:
What is this trying to say? If you want to tell people about how the operators work you should mention the traits, but just saying that Sway is typed and operators only exist when they make sense is redundant.